src/AppInstallerCLICore/Argument.cpp

```diff
@@ -96,7 +96,7 @@ namespace AppInstaller::CLI
         case Args::Type::Wait:
             return Argument{ "wait", NoAlias, Args::Type::Wait, Resource::String::WaitArgumentDescription, ArgumentType::Flag, false };
         case Args::Type::ProductCode:
-            return Argument{ "product-code", NoAlias, Args::Type::ProductCode, Resource::String::ProductCodeArgumentDescription, ArgumentType::Standard, false };
+            return Argument{ "product-code", NoAlias, "pid", Args::Type::ProductCode, Resource::String::ProductCodeArgumentDescription, ArgumentType::Standard, false };

```

Comment:
`pid` usually means "Process ID".  I don't see how it relates to product code.

Reply:
`pid` has multiple meanings. In the terms of a running service, it does typically mean "Process ID". I've seen it used in the application space to refer to "Product ID"

Is there a better alternate name to use here, or should I just remove this one?

Reply:
I find it to be confusing, I would remove it as I don't foresee this being a commonly used argument in need of shortening.